tests/unit/generators/globalConfig_generator/test_globalConfig_generator_init.py

```diff
@@ -0,0 +1,12 @@
+def test___init__html():
```

Comment:
What is the purpose of this test?

Reply:
To ensure that only expected classes and modules exist in `globalConfig_generator`

Reply:
Those modules and classes are internal, I do not expect them to be used by some other program from the outside world so I do not see a lot of value in keeping those tests.